website-page1.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="style.css" rel="stylesheet">
+    <title>Muna Fidow Website</title>
+    <style="background-color: darkolivegreen;"></style>
+  </head>
+  <body>
+    <h1>Muna Fidow Story</h1>
+    <div class="flex-container">
+    <div><img src="images/muna.jpg"WIDTH="300" HEIGHT="300" alt="Muna"/></div>
+    <div><img src="images/goodvibes.jfif"WIDTH="300" HEIGHT="300" alt="good vibes"/></div>
+    </div>
+    <p id="p">I work as an administrator in a  construction company, I like sushi and the colour green. In my spare time I read fiction books, tend to my house plants and watch tv shows. I like Money Heist.
+    <p id="p">I was born and grew up in Italy. I like to vacation where there are bodies of water even though I cant swim. My favourite animal is an Elephant.</p>
+   <div class="flex-container">
+   <div><img src="images/construction.jfif"WIDTH="225" HEIGHT="225" alt="Constructon"/></div>
+   <div><img src="images/houseplants.jfif"WIDTH="225" HEIGHT="225" alt="Houseplants"/></div>
+   <div><img src="images/italy.jfif"WIDTH="225" HEIGHT="225" alt="Italy"/></div>
+   <div><img src="images/netflix.png"WIDTH="225" HEIGHT="225" alt="Netflix"/></div>
+   <div><img src="images/reading.jfif" WIDTH="225" HEIGHT="225" alt="Reading"/></div>
+   <div><img src="images/sushi.jfif" WIDTH="225" HEIGHT="225" alt="Sushi"/></div>
+   <div><img src="images/moneyheist.jfif" WIDTH="225" HEIGHT="225" alt="series"/></div> 
+   <div><img src="images/elephant.jfif" WIDTH="225" HEIGHT="225" alt="elephant"/></div>
```

Comment:
There's a lot of repeated information in these attributes, could this be better as shared information through a CSS rule?

Also, keep an eye on the detail (attributes are lower cased, so instead of WIDTH -> width)